update-server-repo-setup/enable-rmt-repos

```diff
@@ -109,7 +106,30 @@ for repository_info in mirror_info:
             all_repo_ids.append(str(repository_id))
 
 if all_repo_ids:
-    os.system('rmt-cli repos enable %s' % ' '.join(all_repo_ids))
+    chunk_cnt = 0
+    status = 0
+    last = False
+    repo_cnt = 30
+    # Chunk the numbers of repos we enable at one time into 30
+    # 30 is an arbitrary number
+    while 1:
+        if len(all_repo_ids) < 30:
+            enable_repos = all_repo_ids
+            last = True
+        else:
+            if chunk_cnt*repo_cnt+repo_cnt < len(all_repo_ids):
```

Comment:
maybe `chunk_cnt*repo_cnt` could be a variable with a explicit/clear name ?
as it is used few times in the code